R/Orders.R

```diff
@@ -75,40 +75,47 @@ orders <-
     status <- match_letters(status, o = "open", c = "closed", a = "all")
     # check if id
     .is_id <- is_id(symbol_id)
-    if (.is_id) {
+    if (isTRUE(.is_id)) {
       # if it's an order_id
       .o_id <- symbol_id
+      .url <- get_url(
+        path = c("orders", .o_id),
+        query = list(nested = nested),
+        live = live
+      )
+    } else if (isTRUE(client_order_id)) {
+      # if it's a client order id
+      .url <- get_url(
+        path = "orders:by_client_order_id", 
+        query = list(client_order_id = symbol_id), 
+        live = live
+      )
     } else {
-      # if its a ticker symbol
-      .o_id <- NULL
+      # if it's a ticker symbol(s)
       symbol_id <- toupper(symbol_id)
+      if(length(symbol_id) > 1) paste(symbol_id, collapse = ',')
+      .url <- get_url(
+        path = "orders",
+        query = list(
+          status = status,
+          limit = limit,
+          after = after,
+          until = until,
+          direction = direction,
+          nested = nested
+        ),

```

Comment:
We're not passing the `symbols` argument here though, which needs to happen if we're going to remove the `filter` statement that was on line 110